configure

```diff
@@ -30,10 +30,15 @@ WITH_DOCS=1
 ERLANG_MD5="false"
 SKIP_DEPS=0
 
+run_erlang() {
+    erl -noshell -eval "$1" -eval "halt()."
+}
+
 COUCHDB_USER="$(whoami 2>/dev/null || echo couchdb)"
 SM_VSN=${SM_VSN:-"91"}
 ARCH="$(uname -m)"
-ERLANG_VER="$(erl -eval 'io:put_chars(erlang:system_info(otp_release)), halt().' -noshell)"
+ERLANG_VER="$(run_erlang 'io:put_chars(erlang:system_info(otp_release)).')"
+ERLANG_OS="$(run_erlang 'case os:type() of {OS, _} -> io:format("~s~n", [OS]) end.')"
```

Comment:
shell scripts usually use `uname` for this type of stuff. Any particular reason why you chose this? — There is nothing wrong, just curious.

Reply:
The reason for this twofold: I wanted to be consistent with `rebar` / Erlang runtime (I would consider `unix` what it thinks to be UNIX) and avoid reimplementing the same logic in shell as well.